ui/src/components/QRDeposit.tsx

```
@@ -226,34 +230,135 @@ const QRDeposit: React.FC = () => {
-            } else {
-                setError("Failed to create deposit session");
+
+            try {
+                setIsBitcoinLoading(true);
+                const response = await axios.post(`${BITCOIN_PAYMENTS}/invoices`, payload, config);
+                setIsBitcoinLoading(false);
+                console.log("🔷 QRDeposit: Bitcoin payment response:", response.data);
+
+                // Navigate to the payment URL
+                if (response.data && response.data.checkoutLink) {
+                    window.location.href = response.data.checkoutLink;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)